packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -684,6 +687,8 @@ class _LineItemState extends State<LineItem> {
           if (response is! InstanceRef) return;
           final variable = Variable.fromRef(response);
           await _debuggerController.buildVariablesTree(variable);
+          if (_hasMouseExited) return;
```

Comment:
we should do more.  _onHoverExit only gets called if you changed the line. If the user moved more than a couple pixels within the same line, this hover is also obsolete.
we need to verify that we haven't gotten a mouseMove to a different location within the line as opposed to an exit from the line.

Reply:
We already handle that scenario. If the hoverWord is the same we don't change the hover card just like in Chrome.